src/code/getSpeciesSteadyState.R

```diff
@@ -0,0 +1,119 @@
+
+#' Get the steady-state values of species. The steady-state is considered to be the last value of the simulation with sufficiently
+#' long simulation time, i.e., where the rates of the processes do not change.
+#'
+#' @param speciesNames Vector of species names which steady-state values will be returned. By default, values of all species are returned. This also includes
+#' values of "state variable" parameters, as these are treated like species.
+#' @param steadyStateTime Simulation time (minites). Must be long enough for system to reach a steady-state. Default is 1000 minutes.
```

Comment:
I would not define any **default** for the simulation time. It must be in the responsibility of a user to set the time properly.

Reply:
Agree.